cmake/std/atdm/README.md

```diff
@@ -1050,70 +1050,116 @@ $ bsub -x -Is -n 20 \
 
 ```bash
 $ cd <some_build_dir>/
 
+# Load env and configure on the login node
 $ source $TRILINOS_DIR/cmake/std/atdm/load-env.sh cuda-debug
-
 $ cmake -GNinja \
   -DTrilinos_CONFIGURE_OPTIONS_FILE:STRING=cmake/std/atdm/ATDMDevEnv.cmake \
   -DTrilinos_ENABLE_TESTS=ON \
   -DTrilinos_ENABLE_Kokkos=ON \
   $TRILINOS_DIR
 
-$ make NP=20
-```
+# Get a node allocation and log on to launch node
+$ bsub -J <job-name> -W 6:00 -Is bash
 
-You may run the above commands from an interactive bsub session as well using:
+# Build on the allocated compute node 
+$ lrun -n 1 make NP=32
 
-```bash
-$ bsub -J <YOUR_JOB_NAME> -W 4:00 -Is bash
+# Run the test suite from the launch node
+$ ctest -j4
 ```
 
-CTest runs everything using the `jsrun` command. You must run jsrun from a
-compute node which can be acquired using the above bsub command.
+CTest runs everything using the `jsrun` command.  You must run jsrun from the
+launch node allocated using `bsub`.  That is why the raw `ctest` command is
+run on the launch node.
 
-Once you're on a compute node, you can run ctest. For example:
+One can also build directly from the login node on a compute node using:
+
+```
+$ lalloc 1 -W 4:00 make NP=32
+```
+
+One can also directly run the test suite
+from the login node using:
 
 ```bash
-$ ctest -j4
+$ busub -Is -W 2:00 ctest -j4
 ```
 
-The MPI test exectuables are run by a wrapper script `trilinos_jsrun` which
-calls the `jsrun` command which modifies the input arguments to accommodate
-the MPI test suite in Trilinos (see the implementation of the script
-`trilinos_jsrun` for details).  By default, the script `trilinos_jsrun` will
-set `export TPETRA_ASSUME_CUDA_AWARE_MPI=0` if `TPETRA_ASSUME_CUDA_AWARE_MPI`
-is unset in the environment.  Therefore, by default, the tests are run without
-CUDA-aware MPI on this system.
+The advantage of creating a node allocation first using `bsub -Is bash` and
+loging into the launch node is that your builds and test runs from there are
+completely interactive since you don't need to wait for a node allocation.
+
+The MPI test tests in Trilinos are actually run by a wrapper script
+`trilinos_jsrun` which calls the `jsrun` command and modifies the input
+arguments to accommodate the MPI test suite in Trilinos (see the
+implementation of the script `trilinos_jsrun` for details on what it does).
+By default, the script `trilinos_jsrun` will set `export
+TPETRA_ASSUME_CUDA_AWARE_MPI=0` if `TPETRA_ASSUME_CUDA_AWARE_MPI` is unset in
+the environment.  Therefore, by default, the tests are run without CUDA-aware
+MPI on this system.
 
 To explicitly **disable CUDA-aware MPI** when running the test suite, set the
-environment variable:
+environment variable and run as:
 
 ```bash
 $ export TPETRA_ASSUME_CUDA_AWARE_MPI=0
-$ ctest -j4
+$ lrun -n 1 ctest -j4
 ```
 
-and to explicitly **enable CUDA-aware MPI** when running the test suite set:
+To explicitly **enable CUDA-aware MPI** when running the test suite set run
```

Comment:
"To explicitly **enable CUDA-aware MPI** when running the test suite, from the launch node run"